pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -113,7 +113,7 @@ func TestUserSignupWithManualApprovalApproved(t *testing.T) {
 			UID:       types.UID(uuid.NewV4().String()),
 		},
 		Spec: v1alpha1.UserSignupSpec{
-			UserID:            "foo",
+			Username:          "foo",
 			CompliantUsername: "foo",
```

Comment:
Let's use different ComplaintUserName (so it doesn't equal to Username).

Reply:
Fixed all instances of duplicate values.